pyodata/v2/model.py

```diff
@@ -361,6 +362,37 @@ def from_literal(self, value):
         return matches.group(1)
 
 
+def ms_since_epoch_to_datetime(value, tzinfo):
```

Comment:
what the "ms" prefix stands for?

Reply:
milliseconds - but that made the function name too long (>30 characters), and pylint complained. Will add a comment.

Reply:
Oh, obviously... anyway, docstring will help :)